### PR TITLE
VIN Validation + Dark Mode

### PR DESCRIPTION
**Pull request by sloan-seamus**
VINs are now validated before entering them. For testing, seventeen 1's works.

### CODE REVIEW

**Comment by sloan-seamus** [comment]
Dark mode coming soon.

File: frontend/insurance/src/app/shared/components/new-claim/new-claim.component.ts [hunk 1]
```diff
@@ -9,7 +10,11 @@ import { MiddlewareService } from '../../../core/services/middleware/middleware.
 })
 export class NewClaimComponent {
   appearance = new FormControl();
-  vin = new FormControl('', [Validators.required, Validators.minLength(11)]);
+  vin = new FormControl('', [
+    Validators.required,
+    Validators.minLength(11),
+    control => (this.checkVin(control.value) ? null : { foo: 'OOF' }),
```
Comment (dsifford) [inline frontend/insurance/src/app/shared/components/new-claim/new-claim.component.ts:17]: OOF!

File: frontend/insurance/src/app/shared/components/new-claim/new-claim.component.ts [hunk 2]
```diff
@@ -90,11 +95,17 @@ export class NewClaimComponent {
...
+  checkVin(vin: string): boolean {
+    return validate(vin);
+  }
+
   getErrorMessage() {
     return this.vin.hasError('required')
       ? 'VIN is required.'
       : this.vin.hasError('minlength')
       ? 'VIN must be 11 to 17 characters and numbers.'
+      : this.vin.hasError('foo')
```
Comment (dsifford) [inline frontend/insurance/src/app/shared/components/new-claim/new-claim.component.ts:114]: OOF!

**Review by ej3076 — APPROVED** [review]
Fixing VIN validation

**Pull request merged**